box-share-sdk/src/main/java/com/box/androidsdk/share/usx/activities/BoxCollaborationsActivity.java

```diff
@@ -0,0 +1,94 @@
+package com.box.androidsdk.share.usx.activities;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentTransaction;
+
+import com.box.androidsdk.content.models.BoxCollaborationItem;
+import com.box.androidsdk.content.models.BoxIteratorCollaborations;
+import com.box.androidsdk.content.models.BoxSession;
+import com.box.androidsdk.content.utils.SdkUtils;
+import com.box.androidsdk.share.CollaborationUtils;
+import com.box.androidsdk.share.R;
+import com.box.androidsdk.share.usx.fragments.CollaborationsFragment;
+import com.box.androidsdk.share.usx.fragments.CollaboratorsRolesFragment;
+
+/**
+ * Activity used to show and modify the collaborations of an item. The intent to launch this activity can be retrieved via the static getLaunchIntent method
+ */
+public class BoxCollaborationsActivity extends BoxActivity {
+
+    protected static final String TAG = BoxCollaborationsActivity.class.getName();
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.usx_activity_collaborations);
+        initToolbar();
+
+        if (baseShareVM.getShareItem() == null || baseShareVM.getShareItem().getType() == null || !(baseShareVM.getShareItem() instanceof BoxCollaborationItem)) {
```

Comment:
break this line to make it easy to [read](http://google.github.io/styleguide/javaguide.html#s4.5.1-line-wrapping-where-to-break)